util/fipstools/acvp/acvptool/test/tests.json

```diff
@@ -12,6 +12,7 @@
 {"Wrapper": "modulewrapper", "In": "vectors/CMAC-AES.bz2", "Out": "expected/CMAC-AES.bz2"},
 {"Wrapper": "modulewrapper", "In": "vectors/ctrDRBG.bz2", "Out": "expected/ctrDRBG.bz2"},
 {"Wrapper": "modulewrapper", "In": "vectors/ECDSA.bz2", "Out": "expected/ECDSA.bz2"},
+{"Wrapper": "modulewrapper", "In": "vectors/ECDSA-KeyGen.bz2"},
```

Comment:
Do you know why we didn't have this file before?

Reply:
I do not, either it was skipped because there is no way to test expectations/known answers for randomly generated keys (thus why there is no `Out` specified), or it was just a miss.